### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by KrisThielemans**
- Make separate README for PyPI
- Add `pip install` instructions

### CODE REVIEW

**Comment by KrisThielemans** [comment]
Once this is merged, I'll release v0.2.0

**Review by casperdcl — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,8 @@
 /cpp/build/
 /python/petsird/
 __pycache__/
+build/
```
[the remark this answers is not in the document]
Comment (KrisThielemans) [reply .gitignore:5]: I had problems when doing the `pip install python`, giving me a build directory. It all depends on where people execute it (as we don't do it ourselves), so I'd rather keep this in.

[the remark this answers is not in the document]
Comment (casperdcl) [reply .gitignore:5]: FYI `pip install python` probably is different from `pip install ./python`

[the remark this answers is not in the document]
Comment (casperdcl) [reply .gitignore:5]: you also probably want to remove the redundant `/cpp/build/` line if you're dangerous enough to add `build/`

[the remark this answers is not in the document]
Comment (KrisThielemans) [reply .gitignore:5]:
```suggestion
/python/build/
```

File: python/README.md [hunk 2]
```diff
@@ -1,6 +1,33 @@
...
+
+## Pre-installation
+
+You will need the Python files generated by yardl. You have a few options.
+
+### Use PyPI
+
+```sh
+python3 -m venv ~/petsirdenv
+source ~/petsirdenv/bin/activate
```
[the remark this answers is not in the document]
Comment (KrisThielemans) [reply python/README.md:13]: I think I'll leave this in for now. It helps me :-)

File: python/README.pypi.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# PETSIRD Python SDK
+
+The [Emission Tomography Standardization Initiative (ETSI)](https://etsinitiative.org/)
+is working towards establishing a standard for PET Raw Data, called PETSIRD ("PET ETSI Raw Data").
```
[the remark this answers is not in the document]
Comment (KrisThielemans) [reply python/README.pypi.md:4]:
```suggestion
is working towards establishing a standard for PET Raw Data, called PETSIRD ("Positron Emission Tomography Standardization Initiative Raw Data").
```

File: python/README.pypi.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# PETSIRD Python SDK
+
+The [Emission Tomography Standardization Initiative (ETSI)](https://etsinitiative.org/)
+is working towards establishing a standard for PET Raw Data, called PETSIRD ("Positron Emission Tomography Standardization Initiative Raw Data").
```
Comment (casperdcl) [inline python/README.pypi.md:4]: I wish we could standardise on en-GB :sweat_smile:

**Review by casperdcl — APPROVED** [review]

**Pull request merged**